### DISCUSSION TITLE
Future ready talent there is so much to learn in so less time!!

### DISCUSSION THREAD

**Discussion by unidoveguy** [github/india#262] [Students]
Hi all. I have attended the orientation and it seems like they are giving us a huge opportunity to learn and understand the Microsoft Azure cloud platform and its tools and services. But my main concern is how are we gonna able to learn 200+ hours worth of content within just 30 something days (mind you we have to submit the project before 13th Nov) that means every day we have to complete almost 6+ hours of course content. This is like doing full-time lectures all week every week that too along with weekends included.

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
